src/test/config.ts

```diff
@@ -16,7 +16,7 @@ export const config = {
     access: 'idam-user-dashboard--access'
   },
   TestHeadlessBrowser: process.env.TEST_HEADLESS ? process.env.TEST_HEADLESS === 'true' : true,
-  WaitForTimeout: 60000,
+  WaitForTimeout: 600,
```

Comment:
This value is in seconds, not milliseconds, and there is a warning in the log about it. Ten minutes seems more than adequate for a timeout.